src/MagicBall/InputWrapper.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```

Comment:
name files with lowercase like - inputWrapper.js
and component with Upper like - InputWrapper